ntoskrnl/mm/section.c

```diff
@@ -1689,7 +1698,7 @@ MmNotPresentFaultSectionView(PMMSUPPORT AddressSpace,
          * Just map the desired physical page
          */
         Page = (PFN_NUMBER)(Offset.QuadPart >> PAGE_SHIFT);
-        Status = MmCreateVirtualMappingUnsafe(Process,
+        Status = MmCreatePhysicalMapping(Process,
                                               PAddress,
                                               Region->Protect,
                                               Page);
```

Comment:
```suggestion
                                         Page);
```

Reply:
Hint: you can mark multiple lines and suggest a correction for all of them in one comment.